polling/infrequent/activities.py

```diff
@@ -10,10 +10,12 @@ class ComposeGreetingInput:
     greeting: str
     name: str
 
+class ComposeGreeting:
+    def __init__(self, test_service: TestService = None):
```

Comment:
I think this has to be `Optional` to accept a default of `None`. But probably don't even need parameter, just instantiate in constructor always.